docs/src/01_introduction_and_goals.adoc

```diff
@@ -26,22 +26,21 @@ See the complete functional requirements in the xref:#section-annex[Annex] of th
 |===
 |Goal|Description
 | Functional suitability | The system shall fulfill its intended purpose effectively and efficiently, allowing users to register, log in, play the quiz, and access their user statistics.
-| Security | The system must prioritize user data security. It must implement robust authentication mechanisms for user registration and login. The API access points for user information and generated questions must be secured with proper authorization. 
-| Reliability | The system should be reliable in generating questions from Wikidata, ensuring that questions are accurate and diverse. The system must handle user registrations, logins, and game data storage without errors.
-| Availability | The system must be available 99.99% of the time a user tries to access it.
-| Maintainability | The system must be designed and implemented in a way that facilitates easy maintenance and updates.
-| Performance efficiency | The system must deliver optimal performance, ensuring responsive interactions for users. The automatic generation of questions from Wikidata and the real-time gameplay must be efficient. The system must handle N concurrent users.
-| Usability | The system must provide a user-friendly interface, making it easy for users to register, log in, and play the game. The system learning time for a user should be less than 4 hours.
-| Compatibility | The system must be compatible with various web browsers and devices, ensuring a seamless experience for users regardless of their choice of platform. It has to be well-optimized for different screen sizes and functionalities.
-| Transferability | The system must allow for easy transfer of user data and game-related information through its APIs.
+| Reliability | The system should be reliable in generating questions from Wikidata, ensuring that questions are accurate and diverse. The system shall handle user registrations, logins, and game data storage without errors.
+| Availability | The system shall be available 99.99% of the time a user tries to access it.
```

Comment:
I agree with reducing the availability as I believe that it is too high.
About the glossary I think we can all discuss which terms should be defined.

Reply:
I agree with lowering the number; I think 99.99% is too big of a number (99% means an estimated downtime of a little above three days and a half). Also, consider mentioning this would be downtime not caused by the hosting service, as we do not have much control over that.

Reply:
I also believe 99.99% is way too much for our capability. In my opinion 99% is a good goal for us.